osquery/tables/system/magic.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <stdio.h>
+#include <magic.h>
+
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+QueryData genMagicData(QueryContext& context) {
+  QueryData results;
+  magic_t magic_cookie;
+
+  // No default flags
+  magic_cookie = magic_open(MAGIC_NONE);
+
+  if (!magic_cookie) {
+    VLOG(1) << "Unable to initialize magic library.";
```

Comment:
nit, remove the `.` here.

FYI most logs don't end in `.` for some reason? 
```
» grep -rn "LOG(" . | grep "\.\"" | wc -l
      15
» grep -rn "LOG(" . | wc -l
     190
```

Those 15 have arrest warrants posted.